doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -0,0 +1,74 @@
+:orphan:
+
+.. _rendering-documentation:
+
+===================================
+Rendering Documentation with Sphinx
+===================================
+
+SciPy docstrings are rendered to html using `Sphinx`_. Writing
+docstrings is covered in the `numpydoc docstring guide`_; this document
+explains how to check that docstrings render properly.
+
+*For a video walkthrough, please see* \ `Rendering SciPy Documentation
+with Sphinx`_ \ *.*
+
+.. _rendering-documentation-locally:
+
+Rendering Documentation Locally
+-------------------------------
+
+To render the documentation on your own machine (macOS or Linux):
+
+#. Install `Sphinx`_ or ensure that your installation is up to date. For example, if you're using the Anaconda distribution of Python, enter in a terminal window ``conda install sphinx`` or ``conda update sphinx``.
+#. In a terminal window, browse to the ``scipy/doc`` directory. Note the
+   presence of the file ``Makefile``.
+#. If this is your first time building the docs, execute ``git submodule
+   init``. After you’ve initialized for the first time, enter ``git submodule
+   update`` instead. Some of the documentation theme files are not distributed
+   with the main ``scipy`` repository; this keeps them up to date with
+   `git submodules`_.
+#. Enter ``make html-scipyorg``. This uses the `Make build automation tool`_
+   to execute the documentation build instructions from the ``Makefile``.
+   This can take a while the first time, but subsequent documentation builds
+   are typically much faster. *Note: If you use a virtual environment for
+   development, activate it first.*
+#. View the documentation in ``scipy/doc/build/html-scipyorg``. You can start
+   with ``index.html`` and browse, or you can jump straight to the file you’re
+   interested in.
+
+.. note::
+
+   Changes to certain documents do not take effect when Sphinx documentation
+   is rebuilt. In this case, you can build from scratch by deleting the
+   ``scipy/doc/build`` directory, then building again.
+
+.. _rendering-documentation-cloud:
+
+Checking Documentation on the Cloud
+-----------------------------------
+
+Once a PR is opened, you can check that documentation renders correctly
+on the cloud.
+
+#. At the bottom of the PR, select “Show all Checks”.
+#. Next to “ci/circleci: build_docs”, select “Details”.
+#. Find the CircleCI job number. This is the 4-5 digit number in the URL of
```

Comment:
This does not seem necessary. When I log in with a GitHub user with no SciPy privileges I see:

![Screenshot from 2019-06-24 15-01-44](https://user-images.githubusercontent.com/2365790/60044893-6a56ab80-9691-11e9-8fae-69db1ca6e76c.png)

Clicking the "Details" link takes me to:

![Screenshot from 2019-06-24 15-02-16](https://user-images.githubusercontent.com/2365790/60044987-a427b200-9691-11e9-9c33-1cac8bfd6736.png)

If it does not, you just need to sign in to CircleCI, authenticated with GitHub, and try again. No need for this manual stuff.

Then the last step is the standard "Artifacts" tab, where you can get to `index.html`.

Reply:
Done.